recipes/ignition-math/all/conanfile.py

```diff
@@ -68,7 +68,10 @@ def requirements(self):
         self.requires("eigen/3.3.9")
 
     def build_requirements(self):
-        self.build_requires("ignition-cmake/2.5.0")
+        if int(self.version.split(".")[1]) <= 7:
+            self.build_requires("ignition-cmake/2.5.0")
+        else:
+            self.build_requires("ignition-cmake/[^2.8.0]")
```

Comment:
version ranges are not currently recommended in conan center (they may affect on reproducibility).
consider setting some precise version, if possible

Reply:
done